src/config/mod.rs

```diff
@@ -1808,10 +1808,7 @@ impl UnifiedReadPoolConfig {
                     .into(),
             );
         }
-        let limit = cmp::max(
```

Comment:
Why remove the check?

Reply:
If you mean why do I remove the comparsion. The reason is `cpu_cores_quota() * 10` is guaranteed to be bigger than `UNIFIED_READPOOL_MIN_CONCURRENCY`, besides, `UNIFIED_READPOOL_MIN_CONCURRENCY` is not tunable, so there is no need to have this comparison.